target-classifier/TargetClassifier.py

```diff
@@ -71,7 +71,7 @@ def predict(self, X, feature_names, meta):
             matchedEntities = []
 
             # Get the text string that is to be classified.
-            messageText = str(X[0])
```

Comment:
Aren't we removing target classifier? Shouldn't the logic now be:

(question detector) -> (KB search) -> (comprehension)

I think this will require a change to the deployement JSON, docs, and removal of some code right?